schema/descriptor.schema.json

```diff
@@ -257,25 +296,50 @@
                         "id": "http://github.com/boutiques/boutiques-schema/input/max-list-entries",
                         "description": "Specify the maximum number of entries in the list. May only be used with List type inputs.",
                         "type": "number"
+                    },
+                    "absolute-path": {
+                        "id": "http://github.com/boutiques/boutiques-schema/input/absolute-path",
+                        "description": "Specifies that this input must be given as an absolute path. Only specifiable for File type inputs. Cannot be true if relative-path is true.",
+                        "type": "boolean"
+                    },
+                    "relative-path": {
+                        "id": "http://github.com/boutiques/boutiques-schema/input/relative-path",
```

Comment:
Is it necessary to have two mutually-exclusive boolean properties? If you set one to `true`, the other must be `false`. So maybe there could just be one input.

On the other hand, there is the middle case where the image doesn't care about whether the paths are absolute or relative, so neither property would be true. That is a benefit to having two properties. But in that case the tool implementation would have to decide at launch time to either make the path relative or absolute; it wouldn’t matter which one but it does have to set one or the other. But we could guide the tool writers by setting one of the two properties `true` by default.

I guess what I would want is for there to be one boolean property. If `true`, then the paths must be absolute; if `false`, then relative. (Or the other way. Whatever.) And one of those (absolute?) should be the default value if nothing is specified. No guess work by the tool developer.